Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml

```diff
@@ -457,6 +457,18 @@ CanvasColorSpaceEnabled:
     WebCore:
       default: false
 
+CompressionStreamEnabled:
+  type: bool
+  humanReadableName: "Compression Stream API"
+  humanReadableDescription: "Enable Compression Stream API"
+  defaultValue:
+    WebKitLegacy:
+      default: true
```

Comment:
We tend to avoid adding new features to WebKitLegacy to discourage its use, and to avoid triggering bugs that we will have to investigate on a deprecated platform.

Reply:
We also usually start with the setting set to `false` until we are confident the feature is ready to ship.

I would suggest leaving all of these 'false' for now, but make sure to enable the features in WKTR.